hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/qjournal/server/TestJournaledEditsCache.java

```diff
@@ -56,8 +56,11 @@ public class TestJournaledEditsCache {
       PathUtils.getTestDir(TestJournaledEditsCache.class, false);
   private JournaledEditsCache cache;
 
+  private long memory;
+
```

Comment:
this field seems kind of unnecessary -- can we just call `Runtime.getRuntime().maxMemory()` directly on L241? it should at least be a local field